README.md

```diff
@@ -1,2 +1,38 @@
 # django-multiple-forms
 An expansion of django's FormView to handle multiple form submissions
+
+## How To Use
+#### Install the dependency
+```
+pip install django-mulitform-views
+```
+or add `django-multiform-views` to your `requirements.in` file, then run
+```
+pip compile
+pip install -r requirements.txt
+```
+
+#### Usage
+Have your view inherit from ```FormsView```.
+
+Append `multiform_key` to `helper.layout` in the form
+
+Add the fields for `template_name`, `success_url`, etc as you would normally for `FormView`
+
+Add the `form_classes` for each form you want in your view
```

Comment:
-> Instead of the FormView `form_class` property, define the `form_classes` property as a dictionary of...

+ You can instead add form classes dynamically by defining the `get_form_classes` method